package.json

```diff
@@ -60,6 +60,7 @@
     "universal-ga": "^1.2.0"
   },
   "devDependencies": {
+    "@quanle94/innosetup": "^6.0.2",
```

Comment:
What is the reason to not use the `innosetup` package, which seems to be a lot more commonly used than this?

Reply:
Because it is outdated. It is using version 5 and not 6. Several users asked to update but the developer seems not answer, so I search for one fork (there are several, all of them the same fixing precisely the version).
If at some time the original code updates we can change the library.

Reply:
:+1: 

Interesting that the vast majority of users is still using the `innosetup` package.